Testing/Exec/IgnitionDelay/utils/fileio.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
```

Comment:
do we need this? feels like we can just use pandas or something?

Reply:
Is there a memory concern that would encourage us to use pandas over numpy? 
I think not using numpy might make the code a bit more messy, no?

Reply:
Oh sorry, I didn't mean numpy vs pandas. I meant the whole `fileio` thing.